test/route-define-test.js

```diff
@@ -32,13 +33,12 @@ if (typeof steal !== "undefined") {
 		var done = assert.async();
 		mockRoute.start();
 
-
-		canRoute.register("{type}/{id}");
 		var AppState = DefineMap.extend({seal: false},{});
 		var appState = new AppState({type: "dog", id: "4"});
-
 		canRoute.data = appState;
 
+		canRoute.register("{type}/{id}");
+
```

Comment:
@phillipskevin we talked about adding a warning to the `data` setter if routes have already been registered. I’m going to do that in a separate PR.